src/main/java/org/dependencytrack/model/ProjectVersion.java

```diff
@@ -21,11 +21,12 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 
 import java.io.Serializable;
+import java.util.Date;
 import java.util.UUID;
 
 /**
  * Value object holding UUID and version for a project
  */
 @JsonInclude(JsonInclude.Include.NON_NULL)
-public record ProjectVersion(UUID uuid, String version, Boolean active) implements Serializable {
+public record ProjectVersion(UUID uuid, String version, Date inactiveSince) implements Serializable {
```

Comment:
Similar to `Project`, the `boolean active` field should stay for API-facing classes. It can *additionally* include `inactiveSince`.